### ISSUE TITLE
Civil War of the Casters is a bad gamemode

### ISSUE THREAD

**Issue by BomberBro** [d3athrow/vgstation13#27096]
See title, it has some weird elimination objectives versus the other wizards but I've never in my long life ever seen the wizards go for this, sometimes even teaming up despite being from rival factions. It's not like the mode makes it easy for them, if you're the first wizard you literally can't even fight your target for 10 minutes minimum, and by then you may have been killed and the next wizard has to wait 10 minutes. 

As it stands it's just a lethal injection of wizards into a round for the low low price of like 40-45 threat, which is a pretty average threat roll. The rounds at best last 40 minutes and honestly it's not that fun to play unless you're a wizard or a guy who suicided roundstart so you can inhale the three ghost wizard rolls. I dunno how to fix it but I'd start by making the wizard HUDs clearer and spawning them all roundstart so they can actually have a civil war or something.

**Comment by Buchse** [comment]
wizards need to be smacked for not killing each other as it is, it shouldn't be a midround thing at all.

**Comment by ghost** [comment]
Should probably be straight up removed and relegated to a mid-round, round-ending ruleset that requires the total threat to be like 90+ with a cost of 40 and at least 4 candidates to signed up who all get spawned at once. Civil war has always been JUST tier and I'm surprised not enough people have complained about it to get it changed yet.

**Comment by Dacendeth** [comment]
it's really just raging mages number 2.

**Comment by ghost** [comment]
Civil war IS raging mages, in dynamic "raging mages" is the ruleset that fires every time a wizard gets spawned except for roundstart solo ones.

**Comment by gurfan** [comment]
I have a few ideas that I have been thinking of coding, or at least trying to code.

-All wizards spawn at the den roundstart, two per team
-Wizard Teams have their own chat communication
-Wizards start with reduced spell points (70? 60? 80?)
-Slain wizards can have their spells stolen by enemy wizards (or the victor could just be rewarded with extra spell points)
-raise the threat level, obviously

I'm not sure if people would agree with these changes, so please let me know what you think

**Comment by DamianX** [comment]
>-raise the threat level, obviously

it's not obvious to me, can you elaborate?

**Comment by gurfan** [comment]
> > -raise the threat level, obviously
> 
> it's not obvious to me, can you elaborate?

Obvious was a silly thing to say. Raising the threat level to 60 was something that popped into my mind, though I have put much thought into the number. Raising the threat level by itself it probably wouldn't do much besides make it *slightly* less common, which I think most people would by happy with.

**Comment by DamianX** [comment]
Oh you mean increasing the threat COST. Makes sense.

**Comment by boy2mantwicethefam** [comment]
I warned you about the gunk caused by antags but you didn't listen, you can turn almost the entire station into antags but somehow letting wizards gib people is beyond broken

**Comment by Dacendeth** [comment]
Yeah wizards need some incentive besides greentext to actually kill the other wizards because that has been proven to not matter.

**Comment by ghost** [comment]
> 
> 
> > > -raise the threat level, obviously
> > 
> > 
> > it's not obvious to me, can you elaborate?
> 
> Obvious was a silly thing to say. Raising the threat level to 60 was something that popped into my mind, though I have put much thought into the number. Raising the threat level by itself it probably wouldn't do much besides make it _slightly_ less common, which I think most people would by happy with.

So the issue with only raising the threat level is that dynamic doesn't have a ton of requirements to fire rulesets, if the threat is there then it's usually equally likely to get succed up by any qualified ruleset due to dynamic always rolling one at roundstart and weighting not being sufficiently skewed. The preferable solution imo is to make it more difficult to fire by giving it higher requirements besides threat and/or lower weighting, or to make it a midround ruleset like I suggested.

**Comment by BomberBro** [comment]
I like the ideas to change it, although I'd almost prefer if there were three wizards on each side instead of two if they get lower wizbucks for spells. Additionally, they should only get one specific color of robe (red or blue) for each side and maybe a special orange hardsuit instead of the purple one for the red team to differentiate themselves clearly
